cardano-api/src/Cardano/Api/LocalStateQuery.hs

```diff
@@ -149,6 +151,29 @@ queryStakeDistributionFromLocalState network socketPath point = do
       socketPath
       pointAndQuery
 
+queryLocalLedgerState
+  :: blk ~ ShelleyBlock TPraosStandardCrypto
+  => Network
+  -> SocketPath
+  -> Point blk
+  -> ExceptT LocalStateQueryError IO (Either LByteString (Ledger.LedgerState TPraosStandardCrypto))
```

Comment:
Shouldn't we move that `LByteString` error into `LocalStateQueryError`?

Reply:
No, these really are two different things. If CBOR decoding is successful, the return the `LedgerState`. If decoding fails, return the ByteString.

Reply:
Understood now: https://github.com/input-output-hk/cardano-node/pull/1016/files/ba755abfd4b9f4f467629aba4a577dcd59574e08#diff-b548d89cce82358cb320307aed52a300R135